en/guide/device_version.md

```diff
@@ -0,0 +1,43 @@
+## Device Information

```

Comment:
@julianoes I buggered up my merge and picked this new doc up as well - prior to review. I think it is probably OK, but can you please scan to see it is OK?